cli/CommandExecutor.cpp

```diff
@@ -72,26 +80,51 @@ void executeDescribeDatabase(
     }
     max_column_width = std::max(static_cast<int>(relation->getName().length()),
                                     max_column_width);
+    num_blocks.push_back(relation->size_blocks());
+    num_tuples.push_back(PrintToScreen::GetNumTuplesInRelation(
+        reinterpret_cast<const CatalogRelation &>(*relation),
```

Comment:
Do you have to use reinterpret cast? I was hoping that simply ``*relation`` works.